Symfony/CS/Tests/Fixer/Symfony/TrimArraySpacesFixerTest.php

```diff
@@ -0,0 +1,355 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Symfony;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Jared Henderson <jared@netrivet.com>
+ */
+class TrimArraySpacesFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        return array(
+
+            array(
+                "<?php \$foo = array('foo');",
+                "<?php \$foo = array( 'foo' );",
+            ),
+
+            array(
+                "<?php \$foo = ['foo'];",
+                "<?php \$foo = [ 'foo' ];",
```

Comment:
@stof I'm not sure I follow why. This Fixer doesn't change short to long or long to short.  It handles both.  It can run on 5.3+.  I looked at some of the other test cases that involve both types of arrays and they don't split them out into different test methods requiring different versions. Am I missing something? I'm happy to do it if necessary, I just am not understanding why.